lib/Dialect/FIRRTL/Types.cpp

```diff
@@ -296,9 +296,15 @@ int32_t FIRRTLType::getBitWidthOrSentinel() {
 /// asynchronous reset.
 bool FIRRTLType::isResetType() {
   return TypeSwitch<FIRRTLType, bool>(*this)
-      .Case<ResetType, AsyncResetType>([](Type) { return true; })
-      .Case<UIntType>([](UIntType a) { return a.getWidth() == 1; })
-      .Default([](Type) { return false; });
+    .Case<ResetType, AsyncResetType>([](Type) {
```

Comment:
Yep, this was left over from a revision that touched this file, and thus needed to fix the formatting here (I don't think this file has been touched since we added clang-format checks). I will remove this change from this PR, but I can also commit a format-only patch to address this for the future.